.github/workflows/test-phpstan.yml

```diff
@@ -20,16 +20,20 @@ on:
 
 jobs:
   build:
-    name: Analyze code (PHPStan)
+    name: PHP ${{ matrix.php-versions }} Analysis
```

Comment:
```suggestion
    name: PHP ${{ matrix.php-versions }} Static Analysis
```

Reply:
👍 I'm not ready to kick of CI for this again but I will implement this when I am.